05/app.js

```diff
@@ -1,3 +1,30 @@
 console.log('DOM');
 
 const curr = document.querySelector('.js-curr');
+
+const button = document.createElement("button")
+button.textContent = "usuń z koszyka"
+button.style.marginLeft = "5px"
+curr.parentNode.insertBefore(button, curr.nextSibling)
+
+let sibiling = curr.parentNode.firstChild
+while (sibiling) {
+  if (sibiling.nodeType === 1 && sibiling !== curr) {
```

Comment:
Jeśli użyjesz `firElemenentChild` lub `nextElemenetSibling` to wtedy nie musisz sprawdzać czy jest `nodeType` === 1 :)